### PR TITLE
Add new component Tooltip

### PR DESCRIPTION
**Pull request by klzns**
**Ainda falta escrever a documentação.**

image

### CODE REVIEW

File: react/components/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      {children}
+      <div
+        className={`${styles.tooltip} ${
+          visible ? '' : 'dn'
+        } vtex-tooltip__container absolute h-auto mw5 bg-base--inverted pa1 br1 z-5`}>
+        <div className="vtex-tooltip__label bg-base--inverted c-on-base--inverted pa2 t-body">
+          {label}
+        </div>
+        <div
+          className={`${styles.arrow} vtex-tooltip__arrow c-on-base absolute`}
```
Comment (lbebber) [inline react/components/Tooltip/Tooltip.js]: a `c-on-base` nao vai ser necessariamente igual ao `bg-base--inverted`

Comment (lbebber) [reply react/components/Tooltip/Tooltip.js]: só pra ter certeza, essa seria a cor da flecha, isso?

**Comment by cmdalbem** [comment]
@klzns @lbebber o que falta pra mandarmos essa?

**Comment by lbebber** [comment]
The tooltip is to be rewritten—maybe with the same solution as pointed on https://github.com/vtex/styleguide/issues/505—in a way that is less disruptive regarding the DOM

**Comment by cmdalbem** [comment]
@lbebber I see no point in keeping this PR open then, I'll close it.

**Pull request closed without merging**